packages/vite-plugin-cloudflare/src/utils.ts

```diff
@@ -1,4 +1,18 @@
+import * as path from 'node:path';
 import { Request as MiniflareRequest } from 'miniflare';
+import * as vite from 'vite';
+
+export function getOutputDirectory(
```

Comment:
I think that adding unit tests for this utilities would be very very simple and very very useful (both as tests but especially as documentation, practically converting your PR description into tests)

Reply:
[7ec5592](https://github.com/flarelabs-net/vite-plugin-cloudflare/pull/120/commits/7ec55923793372b63b64e49d14e113b46655eb5d)